core/db/curibio/alembic/versions/0b1999fb5ce5_pulse3d_deprecation_date_column.py

```diff
@@ -0,0 +1,28 @@
+"""pulse3d deprecation date column
+
+Revision ID: 0b1999fb5ce5
+Revises: b1c798d03073
+Create Date: 2023-03-10 12:11:27.147725
+
+"""
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = "0b1999fb5ce5"
+down_revision = "b1c798d03073"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # can not remove this new enum value unless deleting it and remaking a new enum type
+    op.execute("ALTER TYPE WorkerState ADD VALUE 'removed'")
```

Comment:
I think there may have been some confusion on the ticket, we don't actually need a `removed` state, a version has been "removed" if it is both deprecated and has reached its EOL, so we only need to add the new column